### ISSUE TITLE
tail -F

### ISSUE THREAD

**Issue by kh000** [gvalkov/tailon-legacy#35]
Is it possible to add tail -F functionality? I need to follow concrete files that I know will be created in the future, however I don't wont to follow whole directory.
Can it be done by simply add new method to CommandControl class in commands.py?
e.g 

```
def tailF(self, n, fn, stdout, stderr, **kw): 
cmd = [self.toolpaths.cmd_tail, '-n', str(n), '-F', fn]
```

**Comment by gvalkov** [comment]
Hello @kh000. That's certainly possible and sounds like a nice feature to have. I'll look into it, but I can't make any promises about when. It's a bit more complicated, since tailon checks to see if all input files exist at start.

For the time being, maybe [wtee](https://github.com/gvalkov/wtee) can be a feasible workaround . You can just pipe any command into it and it'll make it viewable in the browser:

```
tail -F /path/to/file | wtee
```

**Comment by kh000** [comment]
I found out that there is a workaround if tailon is watching symbolic link to non existing (yet) log file. When the log file is created tailon starts streaming it.

**Comment by gvalkov** [comment]
Clever workaround :+1:  I've implemented this feature in 701a32dcbf981f81ad46b61d57a59ec86fd3ab4c. You can just run `tailon -F -f <files>` now and it should work even if the files don't exist yet. I'll probably make a release later today.

Thanks for the nice feature request.
